### ISSUE TITLE
size_format() should be treated as unsafe for output

### ISSUE THREAD

**Issue by johnbillion** [WordPress-Coding-Standards/WordPress-Coding-Standards#449]
The following code does not trigger an `Expected next thing to be an escaping function` warning:

```php
echo size_format( $num );
```

The `size_format()` function uses `number_format_i18n()` internally, which _is_ considered unsafe for output because it outputs localised thousands separators and decimal points.

**Comment by johnbillion** [comment]
It would be nice if `number_format_i18n()` was safe for output, but it's not.

**Comment by JDGrimes** [comment]
[`size_format()` doesn't seem to generate direct output](https://developer.wordpress.org/reference/functions/size_format/#source-code), and it gives this error when preceded by `echo`. Am I missing something?

**Comment by johnbillion** [comment]
Which error are you seeing?

**Comment by JDGrimes** [comment]
```
 2 | ERROR | Expected next thing to be an escaping function (see
   |       | Codex for 'Data Validation'), not 'size_format'
```

**Comment by johnbillion** [comment]
I think I had the `WordPress-Core` standard active when testing this. Works as expected.
